docs/developer-tutorials/local-test-setup.md

```diff
@@ -71,12 +71,15 @@ This setup is deployed with the following security features enabled:
 
    <details>
       <summary>Custom Lifecycle Manager image deployment</summary>
-      If you want to test a custom image of Lifecycle Manager, adapt the `IMG` variable in `Makefile` and run the following:
+      If you want to test a custom image of Lifecycle Manager, run the following:
 
       ```shell
-      make docker-build
-      make docker-push
-      make local-deploy-with-watcher IMG=<image-name>:<image-tag>
+      # build your local image
+      make docker-build IMG=k3d-registry.localhost:5111/<your-image-name>:<your-image-tag>
+      # push it to the local registry
+      make docker-push IMG=k3d-registry.localhost:5111/<your-image-name>:<your-image-tag>
+      # deploy using the image (note the change to port 5000 which is exposed in the cluster)
+      make local-deploy-with-watcher IMG=k3d-registry.localhost:5000/<your-image-name>:<your-image-tag>
```

Comment:
comment: I think we need to provide the registry and especially the port change for it to work

Reply:
:+1: